x/stake/keeper/delegation.go

```diff
@@ -398,9 +398,9 @@ func (k Keeper) Delegate(ctx sdk.Context, delAddr sdk.AccAddress, bondAmt sdk.Co
 
 	// call the appropriate hook if present
 	if found {
-		k.OnDelegationSharesModified(ctx, delAddr, validator.OperatorAddr)
+		k.PostDelegationSharesModified(ctx, delAddr, validator.OperatorAddr)
```

Comment:
This isn't post-modification.

Reply:
All of the hooks were `On` something. I'll have to read through the code in a bit more detail to figure out which ones are `Pre`.

Reply:
Yes, the previous names were (are) wrong / misleading